package.json

```diff
@@ -37,12 +37,14 @@
     ]
   },
   "dependencies": {
+    "cheerio": "^1.0.0-rc.3",
     "express": "^4.16.4",
     "knex": "^0.16.5",
     "path": "^0.12.7",
     "react-router-dom": "^5.0.0",
     "sqlite3": "^4.0.6",
```

Comment:
These testing frameworks (cheerio & supertest) should be dev dependencies.

Reply:
Uninstall and re-install completed.

Reply:
I believe they're both still regular dependencies? Remember to use `--save-dev` or `-D` when you run the install command.